src/client/VZSidebar/Item.tsx

```diff
@@ -23,6 +25,8 @@ export const Item = ({
   handleDeleteClick,
   isDirectory = false,
   isActive = false,
+  renameFileTooltipText='Rename File',
```

Comment:
Nice! For future PRs, please run `npm run prettier` or set up a Prettier extension in your code editor, so that the formatting is consistent (e.g. space around `=`).